packages/neuron-wallet/src/utils/createLoopTask.ts

```diff
@@ -0,0 +1,37 @@
+import { BrowserWindow } from 'electron'
+import env from '../env'
+import SyncBlocksService from '../services/syncBlocks'
+import initConnection from '../typeorm'
+
+// create a background task to sync transactions
+// this task is a renderer process
+const createLoopTask = () => {
+  let loopWindow: BrowserWindow | null = new BrowserWindow({
+    x: 10,
+    y: 10,
+    show: false,
+    webPreferences: {
+      nodeIntegration: false,
+    },
+  })
+
+  loopWindow.loadURL(env.mainURL)
```

Comment:
Must a window (renderer progress) load our main URL?
Can we just load an empty URL or page to avoid unnecessary UI and data loading?

Reply:
I'll check it.

Reply:
Replace `env.mainURL` with an empty file.